workers/unity/Assets/Playground/Scripts/MonoBehaviours/CubeSpawnerInputBehaviour.cs

```diff
@@ -42,26 +43,26 @@ private void OnDeleteSpawnedCubeResponse(CubeSpawner.DeleteSpawnedCube.ReceivedR
             }
         }
 
-        private void Update()
+        private async void Update()
         {
             if (Input.GetKeyDown(KeyCode.Space))
             {
-                SendSpawnCubeCommand();
+                await SendSpawnCubeCommand();
```

Comment:
I don't think you want to await this as it will stop this from reading all other input. You want the handling of the response to be after await.